communications_guidelines.md

```diff
@@ -70,3 +70,6 @@ These guidelines are applicable when acting as a representative of Matplotlib (f
 * Gets over-excited over shiny visualizations - lots of emojis and the like - and encourages folks to share their work. 
 * Highlights various parts of the library, especially the more obscure bits and bobbles. 
 * Acknowleges that it is a sometimes frustrating tangle of bits & bobbles that can confuse even the folks who work on it & signal boosts their confuzzlment. 
+
+## Changes to Guidelines
+Changes to the communications guidelines must be reviewed by the communications lead (currently the community manager) and changes to specific platform guidlines (e.g. twitter, instagram) must be reviewed by the person responsible for that platform when different from the communications lead. If there is no consensus, decisions about community guidelines revert to the communications lead. 
```

Comment:
```suggestion
Changes to the communications guidelines must be reviewed by the communications lead (currently the community manager) and changes to specific platform guidelines (e.g. Twitter, Instagram) must be reviewed by the person responsible for that platform when different from the communications lead. If there is no consensus, decisions about community guidelines revert to the communications lead. 
```

Is it better to remove the `(currently the community manager)`-part and move that information to wherever that is listed? In that way, there is no need to change this document if a separate communications lead is appointed.

Reply:
definitely "first-line" tie breakers!